.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
General ignorance - is this going to conflict with the existing org level dependabot configuration?

Reply:
This file was autogenerated for me. Ill investigate